### DISCUSSION TITLE
Can HF-3c and PBEh-3c be constructed using the D4 correction?

### DISCUSSION THREAD

**Discussion by TyBalduf** [dftd4/dftd4#208] [Q&A] [answered]
I noticed [in the code](https://github.com/dftd4/dftd4/blob/75d502eb5f7170fb1c1677e1772ec8e9c7687d10/src/dftd4/param.f90#L713-L718) that there are D4 parameters defined for HF-3c and PBEh-3c. My understanding was that these particular 3c methods were designed specifically to use the D3 correction. 

Are these D4 parameters meant to replace the D3 correction, so that using this D4 still qualifies as HF-3c/PBEh-3c? Or are these meant to be used on top of or instead of the usual D3 corrections, meaning you could have HF-3c-D4/PBEh-3c-D4 as distinct composite methods?

**Comment by marvinfriede** [comment, marked as answer]
> My understanding was that these particular 3c methods were designed specifically to use the D3 correction.

That is correct. There was no replacement with D4 intended.

Anyway, there are actually no parameters in the code (no corresponding case in [`get_d4eeq_bjatm_parameter`](https://github.com/dftd4/dftd4/blob/75d502eb5f7170fb1c1677e1772ec8e9c7687d10/src/dftd4/param.f90#L139C12-L139C37)). If you try to call the program with something like `--func hf3c`, it will error and tell you that there are no parameters available. I guess this was just a copy and paste oversight from DFT-D3.

This also applies to m062x, m05, m052x, m08hx (case in `get_functional_id` but not in `get_d4eeq_bjatm_parameter`).
